### DISCUSSION TITLE
Support php 7.2 for Jammy

### DISCUSSION THREAD

**Discussion by neddstark** [oerdnj/deb.sury.org#1871] [General]
Hi,

I wasn't able to 100% understand the php versions support. So I will be using php 7.2 on Ubuntu 22 Jammy, and just wanted to make sure that php 7.2 in sury repo will be available until Ubuntu 22 EOL in April 2027, is that correct? And it's updated with critical security patches? I don't want to end up breaking anything in the future when updating the server on routine maintenance.

Thanks a lot for your work, really appreciating your efforts!

**Comment by oerdnj** [comment]
It’s best effort, so yes unless there are some unforeseen circumstances.

https://ubuntu.com/about/release-cycle (**End of Standard Support** matters not End of Life)

**Reply by neddstark** [reply]
Thank you much for the quick response.

In case the 7.2 packages for Jammy are removed from the repo before April 2027, there will be an announcement before they dissapear?

Thanks,

**Comment by neddstark** [comment]
Hi again,

After reading more topics, I understood now that all packages removal is announced properly in advance, that is great.

I made a donation by paypal as a gratitude for your hard work on keeping these repos up and running, and I encourage everybody to support this!

Thank you much,
